### ISSUE TITLE
microdot to mqtt

### ISSUE THREAD

**Issue by ui3o-zone** [donskytech/micropython-web-server-microdot#1]
great work and tutorial! thx for this.
Did you try or consider the microdot webserver to comunicate with a mqtt-broker on an ESP32? just tried something like this and ran out of memory right away. thx for your thoughts on this

**Comment by donskytech** [comment]
No, I have not.  That maybe a good use case.  I will create a post about that later and see what I can do.
